mantidimaging/gui/windows/operations/presenter.py

```diff
@@ -242,10 +242,13 @@ def _post_filter(self, updated_stacks: List[StackVisualiserView], task):
         if task.error is not None:
             # task failed, show why
             self.view.show_error_dialog(f"Operation failed: {task.error}")
-        else:
+        elif use_new_data:
             # Feedback to user
             self.view.clear_notification_dialog()
             self.view.show_operation_completed(self.model.selected_filter.filter_name)
+        else:
+            self.view.clear_notification_dialog()
+            self.view.show_operation_cancelled(self.model.selected_filter.filter_name)
```

Comment:
Could you make a test like `test_post_filter_success` for seeing if this has been called once?